.github/workflows/unittests.yml

```diff
@@ -22,6 +22,36 @@ on:
 
 jobs:
 
+  arrow-backend-test:
+    runs-on: sr404-self-hosted
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v2
+        with:
+          distribution: 'zulu'
+          java-version: '8'
+          java-package: jdk
+      - run: sudo swapoff -a
+      - run: free
+      - run: sudo apt-get update
+      - run: sudo apt-get install -y cmake ccache build-essential
+      - run: sudo apt-get install -y maven
+      - run: sudo apt-get install -y libboost-all-dev libcurl4-openssl-dev
+      - run: sudo apt-get install -y libssl-dev
+      - run: sudo apt-get install -y libz-dev
+      - run: sudo apt-get install -y llvm clang
+      - name: Install Gluten Arrow-backend
+        run: |
+          mvn clean package -Dhttps.proxyHost=child-prc.intel.com -Dhttps.proxyPort=913 -Pbackends-velox -DskipTests -Dbuild_cpp=ON -Dbuild_gazelle_cpp=ON -Dbuild_arrow=ON
```

Comment:
please set the build_gazelle_cpp=OFF in velox test

Reply:
This is for arrow backend, so it should be ON

Reply:
**velox-backend-test:**
    runs-on: sr404-self-hosted
    steps:
I mean in velox, no need to double build arrow

Reply:
fixed in 
9d19a69